File: Softeer/연탄의 크기.py
from sys import stdin

n = int(stdin.readline())
arr = list(map(int, stdin.readline().split()))
arr.sort()
# 어차피 n <= 100

res = 0

for i in range(2, max(arr) + 1):
    cnt = 0
    for j in range(n):
        if arr[j] % i == 0:
            cnt += 1
    res = max(res, cnt)

print(res)

File: Softeer/연탄의 크기.js
var fs = require('fs');
var [n,...arr] = fs.readFileSync('/dev/stdin').toString().trim().split(/\s/);
n = Number(n);
arr = arr.map(Number);

var maxVal = -1;
for(let i = 2; i<=Math.max(...arr);i++){
    cnt = 0;
    arr.forEach((e) => {
        if(e % i === 0){
            cnt++;
        }
    })
    maxVal = Math.max(maxVal, cnt)
}

console.log(maxVal)